templates/card.html

```diff
@@ -29,7 +29,7 @@ <h1>{{ service.organization_name|safe }}</h1>
         <h2>{{ service.name|safe }}</h2>
       {% endif %}
       {% if service.description %}
-        <p>{{ service.description|safe }}</p>
+        <p title="{{ service.description|safe }}">{{ service.description|safe }}</p>
```

Comment:
No, you don't put it twice the same way. You already see the title as the paragraph itself!

Reply:
I need it because I am trimming the big descriptions. So I need a tooltip to show the full description to users.

Reply:
`service.description|safe` (from title) and `service.description|safe` from <p> are not the same?

Reply:
<img width="608" alt="Screenshot 2021-07-22 at 17 00 34" src="https://user-images.githubusercontent.com/1632700/126663688-c5b38d68-c55d-4cb0-aea3-b8ecdb88484f.png">

Reply:
I see the point, but twice the same text is not a good idea according to the almightly search engines.

Reply:
> `service.description|safe` (from title) and `service.description|safe` (from `<p>`) are not the same?

they are the same. I did the trimming thing on CSS.